clang/test/Preprocessor/aarch64-target-features.c

```diff
@@ -671,10 +671,16 @@
 // CHECK-V83-OR-LATER: __ARM_FEATURE_JCVT 1
 // CHECK-V83-OR-LATER: __ARM_FEATURE_PAUTH 1
 // CHECK-V81-OR-LATER: __ARM_FEATURE_QRDMX 1
+// CHECK-BEFORE-V85-NOT: __ARM_FEATURE_BTI 1
 // CHECK-BEFORE-V83-NOT: __ARM_FEATURE_COMPLEX 1
 // CHECK-BEFORE-V83-NOT: __ARM_FEATURE_JCVT 1
 // CHECK-BEFORE-V85-NOT: __ARM_FEATURE_FRINT 1
 
+// Apple A14 and M1 are ArmV8.5-a but do not include BTI.
```

Comment:
Spiritually true, but technically untrue and the technical truth matters here. 

It's 8.4 but with all but one 8.5 extensions.